app/redux/shared_actions/SettingsActionCreators.js

```diff
@@ -0,0 +1,25 @@
+// These exist as a shared action creator because saving values can be done from all over the app
+
+import { 
+	SAVE_END_SET_TIMER, 
+	SET_DEFAULT_METRIC
+} from 'app/ActionTypes';
+
+import * as SetsSelectors from 'app/redux/selectors/SetsSelectors';
+
+export const setDefaultMetric = (metric = 'kgs') => (dispatch, getState) => {
```

Comment:
rename to setDefaultMetric to saveDefaultMetric

Reply:
Done